### PR TITLE
Milestone: setup the project structure

### PR DESCRIPTION
**Pull request by Roseokpe**
In this milestone, I set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules.

### CODE REVIEW

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle:

I know you have made hard work on this project but I cannot proceed review of this pull request because this project should use [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) but you are using GitHub Flow.

_**Please make the above changes and submit a new pull request for a review :+1: This won't be counted as a review because the pull request is invalid :red_circle:**_

**Comment by thecodechaser** [comment]
@Roseokpe check my messages on slack

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <section>
+      <h4>Add Your Score</h4>
+      <form action="">
+        <input type="text" placeholder="Your name" name="name">
+        <input type="text" placeholder="Your score" name="name">
+        <br>
+        <button  class = "bttn" type="submit">Submit</button>
+      </form>
+    </section>
+  </main>
```
Comment (thecodechaser) [inline src/index.html]: - [ ] You did a great job on the design but there are a few issues you need to fix to follow the wireframe design.
 - Please adjust the Leaderboard title to the left side of the screen instead in the center.
 - Please wrap the scoreboard with the border, That will look nice for sure 😄

### Your design

image

### Original design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Roseokpe👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Roseokpe , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**